style.css

```diff
@@ -399,3 +416,156 @@ footer a {
   color: #dbd8d7;
   font-size: 19px;
 }
+
+@media screen and (min-width: 768px) {
+  .logo {
+    margin-left: 140px;
+    border: 1px solid #679;
+  }
+
+  nav {
+    margin-right: 140px;
+  }
+
+  nav > * {
+    margin: 0 15px;
+  }
+
+  .nav-mobile {
+    display: none;
+  }
+
+  .nav-desktop {
+    display: block;
+  }
+
+  .headline {
+    display: grid;
+    height: 870px;
+    grid-template-columns: 140px 560px 1fr;
+    grid-template-rows: repeat(5, 1fr);
+    background-color: #1c1a19;
+    background-image: url('assets/images/illustration-header-mobile.png');
+    background-repeat: no-repeat;
```

Comment:
- [ ] If you take a proper visual view of your design compared to that of the figma template, you'll notice that the background image on the figma design actually suspends from the top right corner, on the other hand, has a little bit of spacing which significantly makes a difference. The reason we emphasize sticking to this design is to get used to meeting project requirements, as you might end up IP building for a client either as a freelance or under a company. please make the c=necceary changes, Great job you did 👏 

# Yours:
![image](https://user-images.githubusercontent.com/50990925/213361118-5332a00b-d2c7-4de2-ba07-966fa4051d99.png)

# Figma :
![image](https://user-images.githubusercontent.com/50990925/213361181-32fe9647-9672-46f8-9c45-180d1d226a71.png)